internal/api_server/middleware/middleware.go

```diff
@@ -57,3 +60,26 @@ func AddEventMetadataToCtx(next http.Handler) http.Handler {
 		next.ServeHTTP(w, r.WithContext(ctx))
 	})
 }
+
+func AddOrgIDToCtx(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		orgIDParam := r.URL.Query().Get("org_id")
+
+		var orgID uuid.UUID
+		if orgIDParam != "" {
+			parsedID, err := uuid.Parse(orgIDParam)
+			if err != nil {
+				http.Error(w, fmt.Sprintf("Invalid org_id parameter: %s", err.Error()), http.StatusBadRequest)
+				return
+			}
+			orgID = parsedID
+		} else {
+			// Fall back to the default organization.
+			orgID = store.NullOrgId
```

Comment:
What's the behavior of the Null org?  Is everyone a member?  If so, I'm concerned that it would lead to information leaks when users forget to pass an orgID, and then their resources can be viewed and modified by everyone.  Do you intend to fix this with RBAC?

Reply:
ACK